views/js/e2e/_helpers/commands/setupCommands.js

```diff
@@ -43,7 +43,7 @@ Cypress.Commands.add('importTestPackage', (fileContent, fileName) => {
     cy.get(setupSelectors.testsPage.fileInput).upload(
         {
             fileContent,
-            fileName,
+            fileName: `${fileName}.zip`,
```

Comment:
Seems to be a Windows thing, but ok, we can go with this for all platforms.

Reply:
Yes, it's a Windows thing and we need it. :(